README.markdown

```diff
@@ -7539,6 +7539,63 @@ This feature was first introduced in the `v0.5.0rc1` release.
 
 [Back to TOC](#nginx-api-for-lua)
 
+ngx.resolve
```

Comment:
This file is automatically generated. See the comment at the beginning of this file for more details. You should edit the `.wiki` file under `doc/` instead and use our toolchain to update this file accordingly.

Reply:
Now, as I have reorganized my implementation, I put resolver.md right near resolver.lua in lua-resty-core module (just like pipe.md, hope this is right).